source/users/custom-metadata.txt

```diff
@@ -296,3 +305,230 @@ Access Custom User Data from a Client Application
 
          Learn how to :ref:`access Custom User Data from a Flutter
          application <flutter-custom-user-data>`.
+
+.. _configure-user-metadata-on-the-backend:
+
+Enable User Metadata 
+--------------------
+
+Atlas App Services can read user metadata from authentication providers. Then, 
+App Services exposes each user's data in a field of their :ref:`user 
+object <user-objects>`. For example, you might want to access a user's name, 
+email, birthday, or gender.
+
+You can configure App Services to request metadata with the access token 
+when users log in. You can access that data from the logged-in user's 
+object with a client SDK.
+
+Configuration
+~~~~~~~~~~~~~
+
+You can define the metadata to request when you configure authentication 
+providers. Specify optional metadata fields that you want to access through 
+the user's account. These metadata fields vary depending on provider.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 80
+
+   * - Provider
+     - Metadata fields
+
+   * - Facebook
+     - ``name``, ``first_name``, ``last_name``, ``picture``, ``gender``, 
+       ``birthday``, ``min_age``, ``max_age``, ``email``
+   
+   * - Google
+     - ``name``, ``first_name``, ``last_name``, ``picture``, ``email``
```

Comment:
nit: i'd keep the bulleted list w/in each of these table cells. easier to read.

```suggestion
     - - ``name``
       - ``first_name``
       - ``last_name``
       - ``picture``
       - ``gender``, 
       - ``birthday``
       - ``min_age``
       - ``max_age``
       - ``email``
   
   * - Google
     - - ``name``
       - ``first_name``
       - ``last_name``
       - ``picture``
       - ``email``
```

if you accept this suggestion, pls make sure that the staging build works correctly. i'm like 90% sure this is correct snooty, but not 100%.